cli/determined_cli/experiment.py

```diff
@@ -963,6 +970,50 @@ def experiment_id_arg(help: str) -> Arg:
             "unarchive experiment",
             [experiment_id_arg("experiment ID to unarchive")],
         ),
+        Cmd(
+            "download",
+            download,
+            "download checkpoints for an experiment",
+            [
+                experiment_id_arg("experiment ID to cancel"),
+                Arg(
+                    "-o",
+                    "--output-dir",
+                    type=str,
+                    default="checkpoints",
+                    help="Desired output directory for the checkpoint.",
```

Comment:
This is a little misleading, since there's one more directory level than this would suggest. I think just changing "checkpoint" to "checkpoints" might help enough, or you could go explicit with another sentence along the lines of "Each downloaded checkpoint will be stored in a subdirectory of this directory named according to the checkpoint's UUID."